### ISSUE TITLE
Can't change resolution

### ISSUE THREAD

**Issue by pistonfish** [Deathspike/crunchyroll.js#33]
The downloaded video is in 1080p even when I set 720p on crunchyroll.

**Comment by th4nkmelater** [comment]
Hello, I am relatively new, I have managed to download episodes. I was wondering how do I get them at a higher quality? They output for me at 480, and I would like them to be 720/1080. Any help would be much appreciated!

**Comment by pistonfish** [comment]
You need a premium Account to download stuff in 720p or 1080p afaik.

**Comment by th4nkmelater** [comment]
Yes, I have a premium account. Does it download auto in HD or do you have to specify?

**Comment by pistonfish** [comment]
Are you logged in? For me the tool was downloading in 1080p no matter the settings. I ended up using some other tool

**Comment by th4nkmelater** [comment]
With the little knowledge, I know of using this program. I was guessing the commands were "crunchy -u USERNAME -p PASSWORD" then you would use "crunchy WEBSITE" it would download them but only at 480.

**Comment by pistonfish** [comment]
I think that I can't really give you an answer. It's six months after I used this tool and it was always downloading in 1080p after logging in. I can't really remember what I was doing but I ended up using another tool.

**Comment by th4nkmelater** [comment]
Could you provide the tool you switched to? Kindest Regards

**Comment by pistonfish** [comment]
I ended up using youtube-dl. You just need to set a cookie in order to login. You should find everything you need via Google. Good Luck!
